### PR TITLE
[PropertyInfo] Add `ConstructorExtractor` documentation for 6.2 (with CPP)

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Follow-up of https://github.com/symfony/symfony-docs/pull/18161

### CODE REVIEW

**Comment by OskarStark** [comment]
Rebase unlocked ✅

File: components/property_info.rst [hunk 1]
```diff
@@ -509,6 +510,31 @@ with the ``property_info`` service in the Symfony Framework::
...
+:class:`Symfony\\Component\\PropertyInfo\\Extractor\\PhpStanExtractor` or
+the :class:`Symfony\\Component\\PropertyInfo\\Extractor\\ReflectionExtractor`
+on the constructor arguments::
+
+    // src/Domain/Foo.php
+    class Foo
+    {
+        public function __construct(private string $bar)
+        {
+        }
```
Comment (OskarStark) [inline components/property_info.rst]:
```suggestion
        public function __construct(
            private string $bar,
        ) {
        }
```

**Comment by alexandre-daubois** [comment]
Rebased! :slightly_smiling_face:

**Comment by OskarStark** [comment]
Thank you @alexandre-daubois.

**Pull request merged**